<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1129</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>业余无线电资格考试练习</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo.ico</normaloff>logo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>业余无线电资格考试练习</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文仿宋</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>题号：</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文仿宋</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>问题：</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>1081</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lab_quest">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>981</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>701</width>
      <height>80</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>按照在省、自治区、直辖市范围内通信所申请设置的业余无线电台，如想要将通信范围扩大至涉及两个以上的省、自治区、直辖市或者涉及境外，或者要到设台地以外进行异地发射操作，须办理下列手续：</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lab_num">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>121</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>华文仿宋</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LK0001</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>1091</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="bnt_ok">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>570</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文宋体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_finish">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>580</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文细黑</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>已完成：100题</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_rate">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>580</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文细黑</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>正确率：100%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>30</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文细黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>C级   题量：1061</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lab_judge_info">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>580</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aharoni</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="intValue" stdset="0">
     <number>60</number>
    </property>
   </widget>
   <widget class="QWidget" name="widget_5" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>681</width>
      <height>360</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_s1">
       <property name="minimumSize">
        <size>
         <width>651</width>
         <height>81</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>新宋体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_s2">
       <property name="minimumSize">
        <size>
         <width>651</width>
         <height>81</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>新宋体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_s3">
       <property name="minimumSize">
        <size>
         <width>651</width>
         <height>81</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>新宋体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_s4">
       <property name="minimumSize">
        <size>
         <width>651</width>
         <height>81</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>新宋体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_6" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>31</width>
      <height>360</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="rbnt_s1">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>12</width>
       <height>81</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>12</width>
       <height>81</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>12</width>
       <height>86</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>123145646</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbnt_s2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>96</y>
       <width>12</width>
       <height>81</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>12</width>
       <height>81</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>12</width>
       <height>86</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>华文仿宋</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbnt_s3">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>183</y>
       <width>12</width>
       <height>81</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>12</width>
       <height>81</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>12</width>
       <height>86</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>华文仿宋</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbnt_s4">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>270</y>
       <width>12</width>
       <height>81</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>12</width>
       <height>81</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>12</width>
       <height>86</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>华文仿宋</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_pic">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>210</y>
      <width>321</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbnt_set">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>570</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1129</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
